public/server/db.json

```diff
@@ -134,6 +134,34 @@
   "pov": {
     "definition": "Point Of View"
   },
+
```

Comment:
You didn't need to add these to the `db.json` folder any longer. I think I'll get rid of the folder soon anyways.

Reply:
Hi, I didn't add these BUT it wouldn't let me continue without fixing the merge conflicts. So, I just took the changes.